src/index.js

```diff
@@ -42,7 +43,11 @@ async function createTestCafe (hostname, port1, port2) {
         getValidPort(port2)
     ]);
 
-    return new TestCafe(hostname, port1, port2);
+    var testcafe = new TestCafe(hostname, port1, port2);
+
+    exitHook(cb => testcafe.close().then(cb));
```

Comment:
I think yes. We should write this in all cases when it can take some time to stop the process